src/axom/sidre/examples/lulesh2/CMakeLists.txt

```diff
@@ -37,11 +37,11 @@ blt_add_executable(
 if(AXOM_ENABLE_TESTS)
     if(ENABLE_MPI)
         blt_add_test(NAME sidre_lulesh2
-                     COMMAND sidre_lulesh2_ex
+                     COMMAND sidre_lulesh2_ex -s 5
                      NUM_MPI_TASKS 1
                      NUM_OMP_THREADS 2 )
     else()
-        blt_add_test(NAME sidre_lulesh2
+        blt_add_test(NAME sidre_lulesh2 -s 5
```

Comment:
Do you intend to merge these changes or were these temporary changes to get things passing in your branch? Should the changes to the `sidre_lulesh` test be reverted to whatever is in develop now?

Reply:
I made this change, and yes, it was intentional.  
Please see the commit message (https://github.com/LLNL/axom/pull/156/commits/9344bbe9dae30bca67943476695306fe5d7ab987)

The default (``30``) was running too slow in debug builds.